webview-ui/src/i18n/locales/en/settings.json

```
@@ -403,9 +403,13 @@
 	},
 	"experimental": {
 		"warning": "⚠️",
+		"autoCondenseContextPercent": {
+			"label": "Threshold to trigger intelligent context condensing",
+			"description": "When the context window reaches this threshold, Roo will automatically condense it."
+		},
 		"AUTO_CONDENSE_CONTEXT": {
-			"name": "Intelligently condense the context window",
-			"description": "Uses an LLM call to summarize the past conversation when the task's context window is almost full, rather than dropping old messages."
+			"name": "Automatically condense the context window",
```

Trying to think of a way to differentiate from the sliding window... maybe this?

```suggestion
			"name": "Automatically trigger intelligent context condensing",
```